config/default/kustomization.yaml

```diff
@@ -51,82 +50,50 @@ components:
   # [GRAFANA] To generate configmap for provision grafana dashboard
   #- ../grafana
 
-replacements:
-# substitutes CERTIFICATE_NAMESPACE, the namespace of the certificate CR
-- source:
-    kind: Certificate
-    name: serving-cert # this name should match the one in certificate.yaml
-    fieldPath: metadata.namespace
-  targets:
-  - select:
-      kind: ValidatingWebhookConfiguration
-      name: validating-webhook-configuration
-    fieldpaths:
-    - metadata.annotations.[cert-manager.io/inject-ca-from]
-    options:
-      delimiter: '/'
-      index: 0
-  - select:
-      kind: CustomResourceDefinition
-    fieldpaths:
-    - metadata.annotations.[cert-manager.io/inject-ca-from]
-    options:
-      delimiter: '/'
-      index: 0
-# substitutes CERTIFICATE_NAME, the name of the certificate CR
-- source:
+transformers:
+- |-
+  apiVersion: builtin
+  kind: NamespaceTransformer
+  metadata:
+    name: adds-resources-to-kcp-system
+    namespace: kcp-system
+  unsetOnly: true
+  setRoleBindingSubjects: allServiceAccounts
+  fieldSpecs:
+  - kind: Namespace
+    path: metadata/name
+    create: true
```

Comment:
NOt sure if I can follow what this does. According to docs:

> will update the metadata/name field of all Namespace objects

But what does this mean in our case here?

Reply:
This one is no longer needed here. I was trying to use this to rename the system namespace but didn't work. will remove it